microsoft-edge/webview2/release-notes.md

```diff
@@ -32,6 +32,159 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+<!-- ====================================================================== -->
+## 1.0.1370.28
+
+Release Date: October 11, 2022
+
+[NuGet package for WebView2 SDK 1.0.1370.28](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1370.28)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 106.0.1370.28 or higher.
+
+### General
+
+#### Promotions
+
+The following items are now stable:
+
+*  The drag and drop API:
+   * `DragEnter`
+   * `DragLeave`
+   * `DragOver`
+   * `Drop`
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2CompositionController.DragLeave Method](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave?view=webview2-dotnet-1.0.1370.28&preserve-view=true#microsoft-web-webview2-core-corewebview2compositioncontroller-dragleave)

```

Comment:
todo: checking now to see if exists for .NET & WinRT:
DragEnter
DragOver
Drop 
if so, will post Markdown here

Reply:
@peiche-jessica @monica-ch - Why no DragEnter, DragOver, & Drop in API Ref docs, for .NET & WinRT, but only for Win32?

Reply:
finding: WinRT: 

this class only lists DragLeave:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2compositioncontroller?view=webview2-winrt-1.0.1370.28&preserve-view=true#dragleave

this interop iface lists all 4 DragFoo:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/interop/icorewebview2compositioncontrollerinterop2?view=webview2-winrt-1.0.1370.28